lib/ueberauth/strategy/cognito.ex

```diff
@@ -256,4 +256,11 @@ defmodule Ueberauth.Strategy.Cognito do
     |> put_private(:cognito_token, nil)
     |> put_private(:cognito_id_token, nil)
   end
+
+  defp option(conn, key) do
```

Comment:
Since this is only used to get the `otp_app` out of the conn, can it be simplified?

```elixir
defp otp_app(conn) do
  default_app = :ueberauth
  if opts = options(conn) do
    Keyword.get(opts, :otp_app, default_app)
  else
    default_app
  end
end
```